content/en/docs/tutorials/hello-minikube.md

```diff
@@ -94,9 +94,11 @@ recommended way to manage the creation and scaling of Pods.
 Pod runs a Container based on the provided Docker image.
 
     ```shell
-    kubectl create deployment hello-node --image=registry.k8s.io/echoserver:1.4
+    kubectl create deployment hello-node --image=registry.k8s.io/echoserver:1.8
     ```
-
+    
+    (for ARM-based deployments, use `--image=registry.k8s.io/echoserver-arm:1.8`)
```

Comment:
Hi @samkass .
The tutorial follows some of the content provided in the Minikube docs (https://minikube.sigs.k8s.io/docs/start/).
If this information is included, you may want to call out the content in a note, something like:

```
{{< note >}}
To deploy the `hello-node` sample code on ARM architectures, substitute the `registry.k8s.io/echoserver-arm:1.8` Docker image in the `kubectl create deployment` command.
{{< /note >}}
```

Reply:
The "note" sounds like a good idea, but I'm not clear where to actually insert it into the document. Also, I don't disagree with the single image suggestion either, but since that doesn't exist yet, it seems like updating the documentation in the meantime is better than nothing. Are you recommending against merging this pull request as-is?

Reply:
Ah, you could insert the note after the code block. Indent the note like the code block.